packages/block-library/src/link-preview/edit.js

```diff
@@ -35,7 +36,34 @@ export default function LinkPreviewEdit( props ) {
 			  },
 	} );
 
-	if ( isEditingUrl ) {
+	useEffect( () => {
+		if ( url && ! title ) {
+			setIsFetching( true );
+			__experimentalFetchUrlData( url )
+				.catch( () => {
+					setHasError( true );
+				} )
```

Comment:
Out of curiosity is putting `.catch` before `.then`  purely syntatical? My understanding is that the Promise will resolve/reject with `then/catch` being called accordingly no matter the order, but interested to know if my mental model is flawed.

Reply:
Order doesn't matter :)

Reply:
Glad I didn't have to completely relearn my mental model 👍